0128-energy-subnetwork.md

```diff
@@ -334,11 +367,22 @@ The ENERGY subnetwork will also help strengthen the IOT subnetwork as the softwa
 
 Applying the governance structure of the Helium community to power systems will increase the numbers of those involved in the influence and buildout of yet another very critical part of our infrastructure. Communication is essential for both sensors and people, but electrical energy is at the core of a modern society. Now is the time to be part of the start in the shift to a new model that is operated by the people - for the people.
 
-### Note on Current Onboarding Fee
+# Stakeholders
+
+This HIP proposes a new subnetwork as such has an overall impact the entire ecosystem. Specifically:
+
+- Helium Foundation - will need to support and provide resources to a new subnetwork.
+- Srcful AB - will need to be the first Service Provider and implement the subnetwork within the bounds of the Helium subnetwork structure.
+- IoT Subnetwork - will be affected by HNT emission sharing, dual mining capabilities, and as a utility for sensor data transportation.
+- Mobile Subnetwork - will be affected by HNT emission sharing.
+
+# Implementation
+
+Onboarding of an ENERGY gateway will be done using the current Helium Wallet App. A specific ENERGY web app will be developed to handle onboarding and configuration of DERs, etc. The ENERGY web app will utilize the Helium Wallet App for authentication and signing of DER-specific settings.
 
-We have implemented a $10 USD onboarding fee in DC due to the current structure of the A variable in the DAO utility score. For future subnetworks, we believe that the main proxy for the utility score should be DC burn for services rather than this onboarding fee. Srcful believes that the A score should be reconsidered in favor of using DC service burn as the proxy for network value, as the current onboarding fee may not be optimal for new subnetworks joining on fair terms.
+Srcful AB and Helium Foundation will be responsible for the implementation of the ENERGY network.
 
-## Definitions
```

Comment:
this seems like it should still be a `##`.

Summary and other sections are level-two titles.

Reply:
Makes sense